docs/components-upgrade.md

```diff
@@ -67,20 +61,20 @@ acs:
 Once these changes are saved run the command below:
 
 ```bash
-ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.0.N-extra-vars.yml"
+ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "acs_play_major_version=23"
```

Comment:
Wouldn't it be even better to ismply document override `acs_play_repository_acs_version` from the CLI extra vars (no file to edit)?